docs/reference/migration/migrate_6_0/rest.asciidoc

```diff
@@ -7,3 +7,7 @@ In previous versions of Elasticsearch, JSON documents were allowed to contain un
 This feature was removed in the 5.x series, but a backwards-compability layer was added via the
 system property `elasticsearch.json.allow_unquoted_field_names`. This backwards-compability layer
 has been removed in Elasticsearch 6.0.0.
+
+==== Analyze API changes
+
+The deprecated request parameters and plain text in request body has been removed, use JSON in request body.
```

Comment:
can we also be more generic when mentioning the request body? We don't support only the json format.